app/views/admin/index.html.erb

```diff
@@ -21,6 +21,7 @@
             <td><%= request.origin_location %></td>
             <td><%= request.destination %></td>
             <td><%= request.item_id %></td>
+            <td><%= request.item_title %></td>
```

Comment:
Also, do you want to add the Requester while you're in here?  There's already a method on the user model that will give you the string you want for SUNet or Name+Email users (we don't handle library ID yet, so idk what we would do there).

https://github.com/sul-dlss/sul-requests/blob/master/app/models/user.rb#L11-L17